src/Shepard/Entity/EntityInterface.php

```diff
@@ -5,7 +5,7 @@
 interface EntityInterface
 {
     /**
-     * @return mixed
+     * @return string/int
```

Comment:
ilyen nincs, hogy string/int, azt egy ilyen vagy-nak || szántam, tehát, ami kell ide az legyen a doc-ban is, vagy vagy, ha string is lehet, akkor legyen az, ha csak int lehet, akkor legyen az.